consensus/vbft/chain_store.go

```diff
@@ -154,6 +142,23 @@ func (self *ChainStore) AddBlock(block *Block) error {
 	return nil
 }
 
+func (self *ChainStore) SubmitBlock(blkNum uint32) error {
+	if blkNum == 0 {
+		return nil
+	}
+	if submitBlk, present := self.pendingBlocks[blkNum]; submitBlk != nil && submitBlk.needSubmitBlock == false && present {
```

Comment:
why `needSubmitBlock == false`?

Reply:
yes,I will modify param name in next commit. needSubmitedBlock

Reply:
rename to `hasSubmitted`

Reply:
ok